### PR TITLE
docs(em): revamp requirements section

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Removed IAM and requirement messages from alerts banners and turned them into text.

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

File: bare-metal/elastic-metal/api-cli/elastic-metal-with-cli.mdx [hunk 1]
```diff
@@ -19,14 +19,13 @@ Scaleway's [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/) p
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/)
-  - You have [generated your API key](/identity-and-access-management/iam/how-to/create-api-keys/)
-  - You have [downloaded and installed the Scaleway CLI](https://github.com/scaleway/scaleway-cli)
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/api-cli/elastic-metal-with-cli.mdx:26]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/activate-remote-access.mdx [hunk 2]
```diff
@@ -16,19 +16,17 @@ categories:
 Remote access is available on most Elastic Metal servers. This option allows you to access the keyboard, video, and mouse of the machine remotely through a KVM-over-IP device.
 Using remote access, you can access the machine for debugging purposes, even if the installed OS is not working properly. It is also possible to install an Operating System on the machine using a remote ISO file.
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/activate-remote-access.mdx:23]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/add-virtual-mac-address.mdx [hunk 3]
```diff
@@ -15,13 +15,12 @@ categories:
...
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key)
-  - You have [created](/bare-metal/elastic-metal/how-to/create-server) and [installed](/bare-metal/elastic-metal/how-to/install-server) an Elastic Metal server
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/add-virtual-mac-address.mdx:22]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/attach-detach-flexible-ip.mdx [hunk 4]
```diff
@@ -15,19 +15,18 @@ categories:
 
 For each of your servers, you can order up to 64 flexible IPs. You can attach and detach any of your [Elastic Metal flexible IPs](/bare-metal/elastic-metal/concepts/#flexible-ip) to and from any of your Elastic Metal servers. The console provides a drag and drop view to facilitate the allocation of your flexible IPs.
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/attach-detach-flexible-ip.mdx:22]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/configure-flexible-ip.mdx [hunk 5]
```diff
@@ -15,14 +15,13 @@ categories:
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key)
-  - You have [created](/bare-metal/elastic-metal/how-to/create-server) and [installed](/bare-metal/elastic-metal/how-to/install-server) an Elastic Metal server
-  - You have [attached a flexible IP address](/bare-metal/elastic-metal/how-to/attach-detach-flexible-ip) to your Elastic Metal server
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/configure-flexible-ip.mdx:22]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/configure-reverse-dns-flexible-ip.mdx [hunk 6]
```diff
@@ -15,14 +15,14 @@ categories:
 
 Reverse DNS, or rDNS, is exactly the opposite of classic forward DNS as we know it. Forward DNS maps a hostname to an IP address. Reverse DNS means we are mapping the IP address to a hostname. This can be very useful, especially if you want to send emails from your Elastic Metal server.
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/configure-reverse-dns-flexible-ip.mdx:22]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/connect-to-server.mdx [hunk 7]
```diff
@@ -15,13 +15,12 @@ categories:
...
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [created your SSH key and added it to your account](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/)
-  - You have [created an Elastic Metal server](/bare-metal/elastic-metal/how-to/create-server)
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/connect-to-server.mdx:22]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/create-server.mdx [hunk 8]
```diff
@@ -15,12 +15,12 @@ categories:
 
 Scaleway [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/) allow you to order dedicated servers on-demand, like cloud Instances. Elastic Metal servers can be used for large workloads, big data and applications that require increased security and dedicated resources.
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/create-server.mdx:22]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/install-server.mdx [hunk 9]
```diff
@@ -18,13 +18,12 @@ Elastic Metal servers can be used for large workloads, big data and applications
...
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key)
-  - You have [created](/bare-metal/elastic-metal/how-to/create-server) an Elastic Metal server
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/install-server.mdx:25]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/order-flexible-ip.mdx [hunk 10]
```diff
@@ -15,17 +15,17 @@ categories:
 
 [Flexible IP addresses](/bare-metal/elastic-metal/concepts/#flexible-ip) are additional IP addresses, available for [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/). They allow you to move an IP from one server to another without changing your whole configuration. Flexible IPs can also be used as additional IP addresses to create virtual machines on your Elastic Metal server.
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/order-flexible-ip.mdx:22]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/reboot-server.mdx [hunk 11]
```diff
@@ -15,13 +15,13 @@ categories:
 
 If your [Elastic Metal server](https://www.scaleway.com/en/elastic-metal/) is no longer reachable from the internet or does not work as expected, you can reboot it from the [Scaleway console](https://console.scaleway.com). The reboot feature launches a power-cycle on the physical hardware and restarts the machine.
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/reboot-server.mdx:22]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/shutdown-server.mdx [hunk 12]
```diff
@@ -15,13 +15,14 @@ categories:
...
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key)
-  - You have [created](/bare-metal/elastic-metal/how-to/create-server) and [installed](/bare-metal/elastic-metal/how-to/install-server) an Elastic Metal server
-</Message>
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/shutdown-server.mdx:23]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/use-rescue-mode.mdx [hunk 13]
```diff
@@ -13,19 +13,19 @@ categories:
...
 <Message type="note">
   This documentation concerns rescue mode for [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/). To use rescue mode on [Instances](https://www.scaleway.com/en/virtual-instances/), refer to the corresponding [documentation](/compute/instances/how-to/use-boot-modes/#how-to-use-rescue-mode).
 </Message>
 
-Rescue mode allows you to boot an [Elastic Metal server](https://www.scaleway.com/en/elastic-metal/) from the network. It downloads a rescue operating system based on [Ubuntu Linux](https://ubuntu.com/) that runs completely in the server RAM. This allows you to perform maintenance and data recovery operations, even if the machine is unreachable in normal mode.
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/use-rescue-mode.mdx:26]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/how-to/view-the-event-logs.mdx [hunk 14]
```diff
@@ -13,13 +13,13 @@ categories:
   - bare-metal
 ---
 
-<Macro id="iam-requirements" />
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/view-the-event-logs.mdx:20]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/quickstart.mdx [hunk 15]
```diff
@@ -13,14 +13,15 @@ categories:
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key)
-</Message>
+## Requirements
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An [SSH key](/console/project/how-to/create-ssh-key/) added to your account
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/quickstart.mdx:24]:
```suggestion
- An [SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/) added to your account
```

File: bare-metal/elastic-metal/api-cli/elastic-metal-with-api.mdx [hunk 16]
```diff
@@ -18,13 +18,12 @@ Our [Elastic Metal server](https://www.scaleway.com/en/elastic-metal/) offer pro
 
 The new deployment in one command feature allows you to speed up the delivery of your servers, as you do not have to wait for it to be delivered to install the server. You can do both in a single API call, and your server is up and running in less than 15 minutes.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (RoRoJ) [inline bare-metal/elastic-metal/api-cli/elastic-metal-with-api.mdx:21]: This PR seems to be missing the new macro and "Before you start" formulation

File: bare-metal/elastic-metal/how-to/attach-detach-flexible-ip.mdx [hunk 17]
```diff
@@ -15,19 +15,18 @@ categories:
 
 For each of your servers, you can order up to 64 flexible IPs. You can attach and detach any of your [Elastic Metal flexible IPs](/bare-metal/elastic-metal/concepts/#flexible-ip) to and from any of your Elastic Metal servers. The console provides a drag and drop view to facilitate the allocation of your flexible IPs.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/attach-detach-flexible-ip.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/attach-detach-flexible-ip.mdx (hunk 4 above)
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/attach-detach-flexible-ip.mdx:22]:
```suggestion
- An [SSH key](/console/project/how-to/create-ssh-key/)
```

File: bare-metal/elastic-metal/how-to/configure-flexible-ip.mdx [hunk 18]
```diff
@@ -15,14 +15,13 @@ categories:
 
 Learn how to configure a [flexible IP address](/bare-metal/elastic-metal/concepts/#flexible-ip) on your Elastic Metal server. The configuration depends on the Linux distribution running on your machine. You can configure your flexible IP on machines running one of [Ubuntu Linux](#how-to-configure-a-flexible-ip-on-ubuntu-linux), [Debian Linux](#how-to-configure-a-flexible-ip-on-debian-linux) or [CentOS](#how-to-configure-a-flexible-ip-on-centos).
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/configure-flexible-ip.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/configure-flexible-ip.mdx (hunk 5 above)
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/configure-flexible-ip.mdx:22]:
```suggestion
- An [SSH key](/console/project/how-to/create-ssh-key/)
```

File: bare-metal/elastic-metal/how-to/configure-flexible-ipv6.mdx [hunk 19]
```diff
@@ -15,17 +15,15 @@ categories:
 
 The configuration of a flexible IPv6 depends on the Linux distribution running on your machine. You can configure your flexible IPv6 on machines running Debian, Ubuntu, or CentOS.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/configure-flexible-ipv6.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx [hunk 20]
```diff
@@ -13,17 +13,16 @@ categories:
   - bare-metal
 ---
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx:16]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/configure-reverse-dns-flexible-ip.mdx [hunk 21]
```diff
@@ -15,14 +15,14 @@ categories:
 
 Reverse DNS, or rDNS, is exactly the opposite of classic forward DNS as we know it. Forward DNS maps a hostname to an IP address. Reverse DNS means we are mapping the IP address to a hostname. This can be very useful, especially if you want to send emails from your Elastic Metal server.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/configure-reverse-dns-flexible-ip.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/connect-to-server.mdx [hunk 22]
```diff
@@ -15,13 +15,12 @@ categories:
 
 This page shows you how to connect to your Scaleway Elastic Metal server via SSH. Thanks to the SSH key you created and uploaded, you can use a terminal application on your local computer to remotely connect to your Elastic Metal server as root. This enables you to use your server as you wish.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/connect-to-server.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/troubleshooting/reset-admin-password-windows-server.mdx [hunk 23]
```diff
@@ -18,12 +18,11 @@ During the installation or reinstallation of an Elastic Metal server with a Wind
...
 If you have lost this password, you can reset it via rescue mode.
 
-<Macro id="iam-requirements" />
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com),
-  - You have [created](/bare-metal/elastic-metal/how-to/create-server) and [installed](/bare-metal/elastic-metal/how-to/install-server) an Elastic Metal server.
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/troubleshooting/reset-admin-password-windows-server.mdx:23]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```

File: bare-metal/elastic-metal/troubleshooting/replace-failed-drive-software-raid.mdx [hunk 24]
```diff
@@ -16,7 +16,9 @@ dates:
 
 Each Elastic Metal server uses a RAID1 configuration after installation from the Scaleway console. If you want to change the RAID configuration of the server, you can modify the RAID array using [rescue mode](/bare-metal/elastic-metal/how-to/use-rescue-mode/).
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/troubleshooting/replace-failed-drive-software-raid.mdx:19]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/troubleshooting/enable-ssh-virtualization-distribution.mdx [hunk 25]
```diff
@@ -20,10 +20,10 @@ Elastic Metal servers support the installation of virtualization solutions Proxm
   Enabling full SSH access to Proxmox or ESXi may cause security vulnerabilities on your installation. Be careful, and take extra precaution when you enable SSH.
 </Message>
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [created](/bare-metal/elastic-metal/how-to/create-server) and [installed](/bare-metal/elastic-metal/how-to/install-server) an Elastic Metal server with Proxmox VE
-</Message>
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/troubleshooting/enable-ssh-virtualization-distribution.mdx:23]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/quickstart.mdx [hunk 26]
```diff
@@ -13,14 +13,15 @@ categories:
...
+Scaleway Elastic Metal are dedicated physical servers you can order on-demand, like Instances. These servers are ideal for large workloads, big data, and applications that require increased security and dedicated resources.
 
-<Macro id="iam-requirements" />
+You can install an OS or other [image](/bare-metal/elastic-metal/concepts/#image) on your Elastic Metal server and connect to it via SSH to use it as you require. You can power off the server when you are not using or delete it from your account once you have finished with it.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH key](/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key)
-</Message>
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/quickstart.mdx:20]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/quickstart.mdx [hunk 27]
```diff
@@ -13,14 +13,15 @@ categories:
...
+
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/quickstart.mdx:22]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
```

File: bare-metal/elastic-metal/quickstart.mdx [hunk 28]
```diff
@@ -13,14 +13,15 @@ categories:
...
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/quickstart.mdx:23]:
```suggestion
```

File: bare-metal/elastic-metal/quickstart.mdx (hunk 15 above)
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/quickstart.mdx:24]:
```suggestion
- An [SSH key](/console/project/how-to/create-ssh-key/)
```

File: bare-metal/elastic-metal/how-to/view-the-event-logs.mdx [hunk 29]
```diff
@@ -13,13 +13,13 @@ categories:
   - bare-metal
 ---
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/view-the-event-logs.mdx:16]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/use-rescue-mode.mdx [hunk 30]
```diff
@@ -13,19 +13,19 @@ categories:
...
 
-<Macro id="iam-requirements" />
+Rescue mode allows you to boot an [Elastic Metal server](https://www.scaleway.com/en/elastic-metal/) from the network. It downloads a rescue operating system based on [Ubuntu Linux](https://ubuntu.com/) that runs completely in the server RAM. This allows you to perform maintenance and data recovery operations, even if the machine is unreachable in normal mode.
 
 <Message type="note">
   This documentation concerns rescue mode for [Elastic Metal servers](https://www.scaleway.com/en/elastic-metal/). To use rescue mode on [Instances](https://www.scaleway.com/en/virtual-instances/), refer to the corresponding [documentation](/compute/instances/how-to/use-boot-modes/#how-to-use-rescue-mode).
 </Message>
 
-Rescue mode allows you to boot an [Elastic Metal server](https://www.scaleway.com/en/elastic-metal/) from the network. It downloads a rescue operating system based on [Ubuntu Linux](https://ubuntu.com/) that runs completely in the server RAM. This allows you to perform maintenance and data recovery operations, even if the machine is unreachable in normal mode.
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/use-rescue-mode.mdx:22]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/use-private-networks.mdx [hunk 31]
```diff
@@ -19,12 +19,11 @@ Each server can be connected to one or several Private Networks, letting you bui
 
 Private Networks allow your Elastic Metal servers to communicate with other Scaleway resources in an isolated and secure network, without the need of being connected to the public internet.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/use-private-networks.mdx:22]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/send-metrics-logs-to-cockpit.mdx [hunk 32]
```diff
@@ -14,14 +14,14 @@ categories:
 ---
 
 Monitoring your dedicated servers with [Scaleway Cockpit](/observability/cockpit/quickstart/) offers detailed information about performance and health, ensuring optimal functionality and proactive issue resolution.
+
 Cockpit provides a user-friendly interface for real-time monitoring of CPU, memory, disk usage, and network activity, enhancing the reliability and performance of bare metal servers.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/send-metrics-logs-to-cockpit.mdx:20]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/how-to/reinstall-server.mdx [hunk 33]
```diff
@@ -15,13 +15,12 @@ categories:
 
 An Elastic Metal server can be reinstalled at any time from the [Scaleway console](https://console.scaleway.com). This can be useful when you want to run a different image on the machine or if you want to have a fresh installation.
 
-<Macro id="iam-requirements" />
+## Requirements
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/reinstall-server.mdx:18]:
```suggestion
<Macro id="requirements" />
```

File: bare-metal/elastic-metal/api-cli/elastic-metal-with-api.mdx (hunk 16 above)
Comment (jcirinosclwy) [reply bare-metal/elastic-metal/api-cli/elastic-metal-with-api.mdx]: There are too many bugs in this PR, so I'll close it and create a new one. :lolcry:
Thanks!

**Pull request closed without merging**